load_balancing.go

```diff
@@ -193,20 +239,63 @@ func (api *API) CreateLoadBalancerMonitor(monitor LoadBalancerMonitor) (LoadBala
 	return r.Result, nil
 }
 
-// ListLoadBalancerMonitors lists load balancer monitors connected to an account.
+// ListLoadBalancerMonitorsInPage lists a single page of load balancer monitors connected to an account.
 //
 // API reference: https://api.cloudflare.com/#load-balancer-monitors-list-monitors
-func (api *API) ListLoadBalancerMonitors() ([]LoadBalancerMonitor, error) {
+func (api *API) ListLoadBalancerMonitorsInPage(pageOpts PaginationOptions) ([]LoadBalancerMonitor, ResultInfo, error) {
```

Comment:
What's the goal of having `xxxxInPage` methods?

Reply:
Allow clients to choose their own pagination strategy/ retrieve a single page of interest? Could also be useful if you just want the ResultInfo from a single call